.github/scripts/backlog-cleanup.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * GitHub Action script for managing issue backlog.
+ * 
+ * Behavior:
+ * - Pull Requests are skipped (only opened issues are processed)
+ * - Skips issues with 'to-be-discussed' label.
+ * - Closes issues with label 'awaiting-response' or without assignees, 
+ *   with a standard closure comment.
+ * - Sends a Friendly Reminder comment to assigned issues without 
+ *   exempt labels that have been inactive for 90+ days.
+ * - Avoids sending duplicate Friendly Reminder comments if one was 
+ *   posted within the last 7 days.
+ * - Moves issues labeled 'questions' to GitHub Discussions
+ */
+
+const dedent = (strings, ...values) => {
+    const raw = typeof strings === 'string' ? [strings] : strings.raw;
+    let result = '';
+    raw.forEach((str, i) => {
+        result += str + (values[i] || '');
+    });
+    const lines = result.split('\n');
+    const minIndent = Math.min(...lines.filter(l => l.trim()).map(l => l.match(/^\s*/)[0].length));
+    return lines.map(l => l.slice(minIndent)).join('\n').trim();
+};
+
+async function fetchAllOpenIssues(github, owner, repo) {
+    const issues = [];
+    let page = 1;
+
+    while (true) {
+        const response = await github.rest.issues.listForRepo({
+            owner,
+            repo,
+            state: 'open',
+            per_page: 100,
+            page,
+        });
+
+        const data = response.data || [];
+        if (data.length === 0) break;
+        const onlyIssues = data.filter(issue => !issue.pull_request);
+        issues.push(...onlyIssues);
+
+        if (data.length < 100) break;
+        page++;
+    }
+    return issues;
+}
+
+
+async function migrateToDiscussion(github, owner, repo, issue) {
+    const discussionCategory = 'Q&A';
+
+    const { data: categories } = await github.rest.discussions.listCategories({
+        owner,
+        repo,
+    });
+
+    const category = categories.find(cat =>
+        cat.name.toLowerCase() === discussionCategory.toLowerCase()
+    );
+
+    if (!category) {
+        throw new Error(`Discussion category '${discussionCategory}' not found.`);
+    }
+
+    const { data: discussion } = await github.rest.discussions.create({
+        owner,
+        repo,
+        title: issue.title,
+        body: `Originally created by @${issue.user.login} in #${issue.number}\n\n---\n\n${issue.body}`,
+        category_id: category.id,
+    });
+
+    await github.rest.issues.createComment({
+        owner,
+        repo,
+        issue_number: issue.number,
+        body: `💬 This issue was moved to [Discussions](${discussion.html_url}) for better visibility.`,
+    });
+
+    await github.rest.issues.update({
+        owner,
+        repo,
+        issue_number: issue.number,
+        state: 'closed',
+    });
+
+    return discussion.html_url;
+}
```

Comment:
I think this will cause the discussion to lose its comments (unlike when clicking `Convert to Discussion`). I don't see any API available to do this though. IDK what would be the best approach here.